fiftyone/core/odm/views.py

```diff
@@ -34,11 +34,3 @@ class SavedViewDocument(Document):
     created_at = DateTimeField()
     last_modified_at = DateTimeField()
     last_loaded_at = DateTimeField()
-
-    def serialize(self):
```

Comment:
@kaixi-wang I don't think this method makes sense here, because converting `_id (ObjectId)` to `id (str)` is a weird quirk of how you chose to build the GraphQL API. It's not something that other users of the `SavedViewDocument` class should ever need to use. Indeed, the intention is that `to_dict()` should always be used to serialize documents.

So, I moved this logic into the server.

Reply:
Removing entirely sounds great. Please do 👍